test/libsolidity/AnalysisFramework.h

```diff
@@ -57,15 +57,17 @@ class AnalysisFramework
 	bool success(std::string const& _source);
 	ErrorList expectError(std::string const& _source, bool _warning = false, bool _allowMultiple = false);
 
-	std::string formatErrors();
-	std::string formatError(Error const& _error);
+	std::string formatErrors() const;
+	std::string formatError(Error const& _error) const;
 
 	static ContractDefinition const* retrieveContractByName(SourceUnit const& _source, std::string const& _name);
 	static FunctionTypePointer retrieveFunctionBySignature(
 		ContractDefinition const& _contract,
 		std::string const& _signature
 	);
 
+	ErrorList filterErrors(ErrorList const& _errorList, bool _includeWarnings) const;
```

Comment:
Can you add a comment about the behaviour with regards to `m_warningsToFilter` and `_includeWarnigs`?